scripts/basicstuff_fuzzer.json

```diff
@@ -0,0 +1,528 @@
+[
+    {
+        "/src/coturn/src/client/ns_turn_msg.c?stun_get_method_str": {
+            "code": "uint16_t stun_get_method_str(const uint8_t *buf, size_t len) {\n {\n  if (!buf || len < 2)\n    return (uint16_t)-1;\n\n  uint16_t tt = nswap16(((const uint16_t *)buf)[0]);\n\n  return (tt & 0x000F) | ((tt & 0x00E0) >> 1) | ((tt & 0x0E00) >> 2) | ((tt & 0x3000) >> 2);\n}",
+            "data": [
+                [
+                    [
+                        [
+                            "288"
+                        ],
+                        [
+                            [
+                                "2049",
+                                "1476395296"
+                            ],
+                            [
+                                "2049",
+                                "1476395296"
+                            ],
+                            [
+                                "2049",
+                                "1476395296"
+                            ]
+                        ]
+                    ]
+                ],
```

Comment:
for data it would be
```json
[
    [input: list[value], outputs: list[list[list]],
    [input: list[value], outputs: list[list[list]],
    [input: list[value], outputs: list[list[list]],
    ...
]
```

the current data seems to have 1 extra pair of `[]`, I will check on and fix this latter today

Reply:
fixed by https://github.com/SecurityLab-UCD/ReportFunctionExecutedPass/pull/15 @harrison4anewbe

Reply:
@harrison4anewbe Can you add the new example here? Also, can you reduce the size of the example. You may contact @EYH0602  for a smaller one.